tests/integration/test_benchmark.py

```diff
@@ -561,7 +561,7 @@ def test_benchmark_single_table_limit_dataset_size():
     results = results.iloc[0]
     assert results['Synthesizer'] == 'GaussianCopulaSynthesizer'
     assert results['Dataset'] == 'adult'
-    assert round(results['Dataset_Size_MB'], 4) == 0.0801
+    assert round(results['Dataset_Size_MB'], 4) in (0.0801, 0.088)
```

Comment:
Why is it possible to have two different dataset sizes?

Reply:
I guess the new value should be `0.088` this is due to the way I changed the `sample` instead of the `head` to be returned when subsampling to match the docstring. This may be actually fragile test...

Reply:
Fixed here: [df3e15e](https://github.com/sdv-dev/SDGym/pull/471/commits/df3e15e2dc38f602342741195718690b60153544)